.circleci/config.yml

```diff
@@ -1511,12 +1511,22 @@ jobs:
           docker_layer_caching: true
       - run:
           name: Run Kontrol Tests
-          command: pnpm test:kontrol
+          command: |
```

Comment:
Would be nice to add some comments documenting this new way proofs are run, e.g. what we are doing, any security assumptions/tradeoffs, etc

Reply:
Where would you like this documented? Happy to put something together, but make sure they're visible to the developers who need to see it.

Reply:
Maybe we add the docs to `packages/contracts-bedrock/test/kontrol/README.md` and then just reference it with a comment in this file?

Reply:
Okay sounds good. I will work on this tomorrow after talking with @raffaele-oplabs